mantidimaging/core/io/utility.py

```diff
@@ -163,21 +166,13 @@ def find_images(sample_dirname: Path,
     return []
 
 
-def find_log(dirname: Path, log_name: str, logger: Logger = None) -> str:
-    """
-
-    :param dirname: The directory in which the sample images were found
-    :param log_name: The log name is typically the directory name of the sample
-    :param logger: The logger that find_log should report back via, should an error occur.
-    :return:
-    """
-    expected_path = dirname / '..'
-    try:
-        return get_file_names(expected_path.absolute(), "txt", prefix=log_name)[0]
-    except RuntimeError:
-        if logger is not None:
-            logger.info(f"Could not find a log file for {log_name} in {dirname}")
-    return ""
+def find_log_for_image(image_file_name: Path) -> Optional[Path]:
+    fg = FilenameGroup.from_file(image_file_name)
```

Comment:
NITPICK: Could we make this variable name more descriptive maybe `filename_group`